build.gradle

```diff
@@ -49,7 +49,7 @@ allprojects {
         }
 
         maven {
-            url = 'https://maven.pkg.github.com/ballerina-platform/ballerina-lang'
+            url = 'https://maven.pkg.github.com/ballerina-platform/*'
```

Comment:
Since there is only one url, we can leave it as it is

Reply:
Updated in e8c9297